app/graphql/mutations/account/grant_site_permission.rb

```diff
@@ -0,0 +1,29 @@
+class Mutations::Account::GrantSitePermission < Mutations::Base
+  argument :account, ID,
+    required: true,
+    description: 'Who to grant permissions to'
+
+  argument :permission, Types::Enum::SitePermission,
+    required: true,
+    description: 'The permission to grant to this user'
+
+  field :id, ID, null: false
+  field :permissions, [Types::Enum::SitePermission], null: false
+
+  def load_account(value)
+    ::User.find(value)
+  end
+
+  def authorized?
+    current_user.permissions.admin?
```

Comment:
Should we be adding this as a pundit scope?

Reply:
I was thinking about it, but I'm still not sure how best to represent it in Pundit.  Pundit's model is very much model-centric vs action-centric :/

Reply:
I think we discussed this on slack and concluded we should just add more pundit methods.